hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/CreateDagVisitor.java

```diff
@@ -106,6 +107,7 @@ public class CreateDagVisitor {
     private final Address localMemberAddress;
     private final QueryParameterMetadata parameterMetadata;
     private final WatermarkKeysAssigner watermarkKeysAssigner;
+    private final WindowSizeGcdCalculator windowSizeGCDCalculator;
```

Comment:
Maybe we should be consistent with letter capitalization? Either ```gcd```/```Gcd``` or ```GCD````.

Reply:
Leftover after renaming, thanks.